examples/grpc_attestation/client/java/BUILD

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright 2021 The Project Oak Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+load("@rules_java//java:defs.bzl", "java_binary")
+
+package(
+    licenses = ["notice"],
+)
+
+java_binary(
+    name = "client",
+    srcs = [
+        "src/Main.java",
+    ],
+    main_class = "com.google.oak.examples.grpc_attestation.client.Main",
+    deps = [
+        "//oak_functions/client/java:client",
```

Comment:
I think it is a bit strange that this example in `examples` has a dependency to `oak_functions`. Could this entire example be moved to `oak_functions/examples`? Alternatively, it might make more sense to have `grpc_attestation` as a top-level crate/folder in `oak`, especially if it is not exclusively for Oak Functions.

Reply:
I'll remove this example, once `grpc_attestation` is merged in Oak Functions.